packages/compiler/src/core/scanner.ts

```diff
@@ -1036,6 +1042,14 @@ export function createScanner(
     return text;
   }
 
+  function getDocTextValue(): string {
+    if (tokenFlags & TokenFlags.Escaped) {
+      return unescapeString(tokenPosition, position);
```

Comment:
We talked and I think we should have a different escape logic for the doc comment to be equivalent to JSDoc

Currently with your implementation it is erroring on invalid ecsape  [See this playground](https://cadlplayground.z22.web.core.windows.net/prs/3375/?c=LyoqIHNvbWUgXGkgZXNjYXBlIFwqLwptb2RlbCBGb28ge30K&e=%40typespec%2Fopenapi3&options=%7B%7D) which is the logic we want for string but this would not be the same behavior as jsdoc(and also potentially a breaking change)

So we think if the backslash doesn't escape anything special it should just ignore and include it as is



```
/** Some \i escaped\*/
```
should result in `Some \i escaped\` string value

Reply:
I've added logic to ignore invalid escape sequences and return them verbatim, is this the direction you'd like to go?

Reply:
Sorry, was on holiday for the last two weeks, I've just added this case as well.